content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -1,8 +1,148 @@
 # DVC Files and Directories
 
 Once initialized in a <abbr>project</abbr>, DVC populates its installation
-directory (`.dvc/`) with the internal files and directories needed for DVC
-operation:
+directory (`.dvc/`) with the
+[internal directories and files](#internal-directories-and-files) needed for DVC
+operation.
+
+Additionally, there are two special kind of files created by certain
+[DVC commands](/doc/command-reference):
+
+- Files ending with the `.dvc` extension are placeholders to version data files
+  and directories. A <abbr>DVC project</abbr> usually has one
+  [`.dvc` file](#dvc-files) per large data file or dataset directory being
+  tracked.
+- The [`dvc.yaml` file](#dvcyaml-files) or _pipeline(s) file_ specifies stages
+  that form the pipeline(s) of a project, and their connections (_dependency
+  graph_ or DAG).
+
+Both use human-friendly YAML schemas, described below. We encourage you to get
+familiar with them so you may edit them freely, as needed. Both type of files
+should be versioned with Git (for Git-enabled <abbr>repositories</abbr>).
+
+## .dvc files
+
+When you add a file or directory to a <abbr>DVC project</abbr> with `dvc add` or
+`dvc import`, a `.dvc` file is created based on the data file name (e.g.
+`data.xml.dvc`). These files contain the information needed to track the data
+with DVC.
+
+They use a simple [YAML](https://yaml.org/) format, meant to be easy to read,
+edit, or even created manually by users. Here is a full sample:
+
+```yaml
+outs:
+  - md5: a304afb96060aad90176268345e10355
+    path: data.xml
+
+# Comments and user metadata are supported.
+meta:
+  name: 'John Doe'
+  email: john@doe.com
+```
+
+`.dvc` files can contain the following two fields:
+
+- `outs`: List of <abbr>outputs</abbr> for this `.dvc` file
+- `deps` (optional): List of <abbr>dependencies</abbr> for this stage, only
+  present when `dvc import` and `dvc import-url` are used.
+- `meta` (manual): Arbitrary information can be added here manually. Any YAML
+  contents can be added. `meta` contents are ignored by DVC, but they can be
+  useful for user processes that read `.dvc` files.
+
+An output entry can consist of these fields:
+
+- `md5`: Hash value for the output file
+- `path`: Path to the output in the <abbr>workspace</abbr>, relative to the
+  location of the `.dvc` file
+- `cache` (optional): Whether or not DVC should cache the output. `true` by
```

Comment:
@efiop @skshetry does cache=false still make sense in DVC 1.0 for .dvc files?

Reply:
It does not (except old-style stagefiles), but I guess it's fine to write it as an `optional` field.

Reply:
Sorry, I'm confused. Does the `cache` field exist in dvc.yaml? Thanks

Reply:
It does, as an attribute of an output. But, isn't this about `.dvc` files?

Reply:
let's deprecate this? what is the meaning for this?

Reply:
> isn't this about .dvc files?

Both .dvc files and dvc.files are specified in this doc. Soon also dvc.yaml and maybe params.yaml

All the special DVC files

> let's deprecate this?

Should we open an issue in the core repo?